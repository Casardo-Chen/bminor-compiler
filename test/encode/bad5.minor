"this\string\has\a\lot\of\slashes"