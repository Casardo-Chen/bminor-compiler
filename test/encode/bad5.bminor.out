Input String    : "this\string\has\a\lot\of\slashes"
encode error: Invalid escape sequence
Invalid String: "this\string\has\a\lot\of\slashes"
